projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -147,7 +148,7 @@ public static function get_initial_state() {
 			'connectUrl'                  => false == $current_user_data['isConnected'] // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
 				? Jetpack::init()->build_connect_url( true, false, false )
 				: '',
-			'dismissedNotices'            => self::get_dismissed_jetpack_notices(),
```

Comment:
Should the `get_dismissed_jetpack_notices` method be removed because it's not used anywhere else?

Or may be add a deprecation notice?

Reply:
Done